### PR TITLE
Changed log error to warning when importing combinations.

### LINKED ISSUE #809: Extra error: "The ODF does not match the combination file"
When I try to import a combination saved from a previous version of ODF, the combination is loaded successfully, but the error message "The ODF does not match the combination file" appears in the log window.

1. I import a combination into the organ, not the organ into the combination. So this message is not precise. ``The combination file does not match the current ODF`` sounds more precisely.
2. Because the combination is loaded anyway, the message should be a warning, not an error.

### PR DESCRIPTION
**Pull request by larspalo**
Resolves #809

### CODE REVIEW

**Comment by larspalo** [comment]
As I tested importing only combinations to changed odfs I only had issues when I had removed a stop from the odf - in such a situation I get an error log message and the combinations won't import. New stops that I had added behaved properly - they were simply not set. Existing but with changes had no issues either.

To correct the import for an odf which has a removed stop, I think we have to change the parsing behaviour (to just ignore a cmb section when it's not present in the odf). But that's perhaps another issue?

Interestingly enough, I didn't have any issues when importing just the settings (with #810 of course) so the problem with removed stops are limited to if existing combination entries are present in the frame generals.

**Comment by oleg68** [comment]
> To correct the import for an odf which has a removed stop, I think we have to change the parsing behaviour (to just ignore a cmb section when it's not present in the odf). But that's perhaps another issue?

Yes, indeed, another issue.

File: src/grandorgue/GrandOrgueFile.cpp [hunk 1]
```diff
@@ -619,13 +619,16 @@ void GrandOrgueFile::LoadCombination(const wxString& file)
 		if (hash != wxEmptyString)
 			if (hash != m_ODFHash)
 			{
-				wxLogWarning(_("The combination file does not exactly match the current ODF."));
+				if (wxMessageBox(_("The combination file does not exactly match the current ODF. Importing it can cause various problems. Should it really be imported?"), _("Import"), wxYES_NO, NULL) == wxYES)
```
Comment (oleg68) [inline src/grandorgue/GrandOrgueFile.cpp:622]: Clicking every time to OK is not good.

Can you display the message box only if the churche name has changed (probably this combinations are from a totally another organ)? 

If the churche name is the same but hash is different then only to log warning?

Comment (larspalo) [reply src/grandorgue/GrandOrgueFile.cpp:622]: Sure, it's certainly possible. I'll fix that tomorrow!

Comment (larspalo) [reply src/grandorgue/GrandOrgueFile.cpp]: I've added a new solution that works better.

**Review by oleg68 — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**